playbooks/common/openshift-cluster/config.yml

```diff
@@ -17,6 +17,24 @@
       - package_version
       - docker_image_availability
       - docker_storage
+  when: ansible_distribution != "Fedora"
+
+- name: Verify Requirements - Fedora
+  hosts: oo_all_hosts
+  roles:
+  - openshift_health_checker
+  vars:
+  - r_openshift_health_checker_playbook_context: install
+  post_tasks:
+  - action: openshift_health_check
+    args:
+      checks:
+      - disk_availability
+      - memory_availability
+      - package_availability
+      - docker_image_availability
+      - docker_storage
+  when: ansible_distribution == "Fedora"
```

Comment:
Neat! Well I don't feel strongly either way. Nothing magical happens if your system doesn't meet the requirements, except that things probably won't run as well as you would like at scale, and Red Hat will consider you unsupported. Running checks and gating the install on it is a balancing act between annoying the user and informing them, so you have to consider whether the information is valuable to enough people to risk annoying the rest of them.